external_deps/build.sh

```diff
@@ -5,34 +5,33 @@
 set -e
 set -u
 
-# Dependencies version. This number must be updated every time the version
-# numbers below change, or packages are added/removed.
+# This should match the DEPS_VERSION in CMakeLists.txt.
+# This is mostly to ensure the path the files end up at if you build deps yourself
+# are the same as the ones when extracting from the downloaded packages.
 DEPS_VERSION=5
```

Comment:
Also I think you want to set this to 6.

Reply:
That can only be done after the new builds are on the server. I want to merge first before doing the final builds.